index.js

```diff
@@ -48,6 +48,7 @@ function send(ctx, path, opts) {
     var maxage = opts.maxage || opts.maxAge || 0;
     var hidden = opts.hidden || false;
     var format = opts.format === false ? false : true;
+    var extensions = Array.isArray(opts.extensions) ? opts.extensions : false;
```

Comment:
Yep. But if there is a invalid extension param in `extensions` option, we should throw an error, not just ignore it silently ?